vips/vip-501/bscmainnet.ts

```diff
@@ -0,0 +1,346 @@
+    jump: "3",
+    kink: "0.45",
+  },
+};
+
+export const wstETHMarket: Market = {
+  vToken: {
+    address: "0xbEC19Bef402C697a7be315d3e59E5F65b89Fa1BB",
+    name: "Venus wstETH (Core)",
+    symbol: "vwstETH_Core",
+    underlying: wstETH,
+    decimals: 8,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: COMPTROLLER_CORE,
+  },
+  riskParameters: {
+    collateralFactor: parseUnits("0.7", 18),
+    liquidationThreshold: parseUnits("0.725", 18),
+    supplyCap: parseUnits("14000", 18),
+    borrowCap: parseUnits("7000", 18),
+    reserveFactor: parseUnits("0.25", 18),
+    protocolSeizeShare: parseUnits("0.05", 18),
+  },
+  initialSupply: {
+    amount: parseUnits("3.208916246034338443", 18),
+    vTokensToBurn: parseUnits("0.03309", 8), // around $100
+    vTokenReceiver: unichainmainnet.VTREASURY,
+  },
+  interestRateModel: {
+    address: "0x5C7D8858a25778d992eE803Ce79F1eff60c1d9D1",
+    base: "0",
+    multiplier: "0.15",
+    jump: "3",
+    kink: "0.45",
+  },
+};
+
+const vip501 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-501 [Unichain] Add weETH and wstETH markets to the Core pool",
+    description: "",
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+
+  return makeProposal(
+    [
+      // <--- weETH Market --->
+      // Oracle config
+      {
+        target: unichainmainnet.REDSTONE_ORACLE,
+        signature: "setTokenConfig((address,address,uint256))",
+        params: [[weETH.address, WEETH_REDSTONE_FEED, STALE_PERIOD_26H]],
+        dstChainId: LzChainId.unichainmainnet,
+      },
+      {
+        target: unichainmainnet.RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3],bool))",
+        params: [
+          [
+            weETH.address,
+            [WEETH_ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero],
+            [true, false, false],
+            false,
+          ],
+        ],
+        dstChainId: LzChainId.unichainmainnet,
+      },
+
+      // Market configurations
+      {
+        target: weETHMarket.vToken.address,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["86400"],
+        dstChainId: LzChainId.unichainmainnet,
+      },
+      {
+        target: unichainmainnet.VTREASURY,
+        signature: "withdrawTreasuryToken(address,uint256,address)",
+        params: [
+          weETHMarket.vToken.underlying.address,
+          weETHMarket.initialSupply.amount,
+          unichainmainnet.NORMAL_TIMELOCK,
+        ],
+        dstChainId: LzChainId.unichainmainnet,
+      },
+      {
+        target: weETH.address,
+        signature: "approve(address,uint256)",
+        params: [unichainmainnet.POOL_REGISTRY, 0],
+        dstChainId: LzChainId.unichainmainnet,
+      },
```

Comment:
We could remove this command, if that allows us to include every command in one VIP